rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2193,6 +2196,47 @@ def test_upgrade_db_27_to_28(user_data_dir):  # pylint: disable=unused-argument
     assert db.get_version() == 28
 
 
+@pytest.mark.parametrize('use_clean_caching_directory', [True])
+def test_upgrade_db_28_to_29(user_data_dir):  # pylint: disable=unused-argument
```

Comment:
as I said don't do an upgrade just for this

Reply:
What would you do here then?